targets/XPlatCppSdk/source/test/TestGSDKApp/TestGSDKApp.cpp

```diff
@@ -0,0 +1,96 @@
+#include "pch.h"
+// Copyright (C) Microsoft Corporation. All rights reserved.
+//
+// This is a small application that demonstrates how to integrate with the GSDK
+// for PlayFab Multiplayer Servers 2.0. To run it, you will need to follow the
+// instructions for Local Debugging and set up our Local Agent:
+// https://api.playfab.com/docs/tutorials/landing-tournaments/multiplayer-servers-2.0/debugging-playfab-multiplayer-platform-integration-locally
```

Comment:
Is this runnable using our functional tests? I am guessing eventually we would want this on github gsdksamples

Reply:
the app isn't a runner, just a quick console app to test the integration, there is currently a c++ runner in our GSDK github though, and yes that one should go in GSDK Samples (I'll make sure to add a task)

---
In reply to: [256916604](https://github.com/PlayFab/SDKGenerator/pull/402#discussion_r256916604) [](ancestors = 256916604)